src/components/tiltak/TiltakFormHeading.tsx

```diff
@@ -1,15 +1,32 @@
 import { BodyLong, Heading } from "@navikt/ds-react";
 import { ReactElement } from "react";
+import { useAudience } from "../../hooks/routeHooks";
+
+const arbeidstakerTexts = {
+  title: "Hva kan gjøre det lettere å jobbe?",
+  body: "Når dere har fått oversikt over arbeidsoppgavene dine, kan dere se på hva slags tilrettelegging det er mulig å tilby.",
+};
+
+const arbeidsgiverTexts = {
+  title: "Hva kan dere gjøre som arbeidsgiver?",
+  body: "",
+};
 
 export const TiltakFormHeading = (): ReactElement => {
```

Comment:
Dersom du velger å gå for en komposisjon-løsning med children som jeg nevnte over, så kan du gjerne også her unngå å bruke audience, men heller sende inn TiltakFormHeading med riktige verdier for SM og AG. Kanskje kan denne komponenten ta inn title og body som props, og så kan du sende de inn på SM og AG page?